lib/config/common.ts

```diff
@@ -165,7 +165,7 @@ export interface RenovateConfig
   prHourlyLimit?: number;
   repoIsOnboarded?: boolean;
 
-  updateType?: UpdateType;
+  matchUpdateTypes?: UpdateType[];
```

Comment:
Sorry, we maybe had a confusion here. The `updateTypes` in `packageRules` should be renamed to `matchUpateTypes`. The `updateType` that's returned from our lookup routine should be renamed `updateTypes`.

Reply:
Ah, sorry 🙃